lib/fog/openstack/requests/compute/evacuate_server.rb

```diff
@@ -3,12 +3,12 @@ module Compute
     class OpenStack
       class Real
         def evacuate_server(server_id, host, on_shared_storage, admin_password = nil)
+          evacuate_hash = {}
```

Comment:
also if the parameters are optional, change also signature of https://github.com/sseago/fog-openstack/blob/evacuate_server_api_fix/lib/fog/openstack/models/compute/server.rb#L305